package.json

```diff
@@ -36,7 +38,7 @@
     "dev:test": "jest --watch",
     "format": "prettier --write .",
     "prepublishOnly": "yarn build",
-    "build": "rm -rf dist && tsc -d",
+    "build": "rm -rf dist && tsc -d && tsc -p tsconfig.esm.json",
```

Comment:
Can you create a tsconifg for `cjs` too please.

Reply:
The default `tsconfig.json` is already for `cjs`. Do you want me to create a `tsconfig.cjs.json`?

Reply:
Yep please

Reply:
ping